src/storage/hummock_test/src/test_utils.rs

```diff
@@ -353,3 +356,86 @@ pub async fn register_test_tables<S: MetaStore>(
     )
     .await;
 }
+
+pub async fn register_test_tables_with_catalog<S: MetaStore>(
+    filter_key_extractor_manager: &FilterKeyExtractorManagerRef,
+    hummock_manager_ref: &HummockManagerRef<S>,
+    tables: &[ProstTable],
+) {
+    update_filter_key_extractor_for_tables(filter_key_extractor_manager, tables);
+    let table_ids = tables.iter().map(|t| t.id).collect_vec();
+    register_table_ids_to_compaction_group(
+        hummock_manager_ref,
+        &table_ids,
+        StaticCompactionGroupId::StateDefault.into(),
+    )
+    .await;
+}
+
+#[allow(dead_code)]
```

Comment:
`#[allow(dead_code)]` why do we need this flag for HummockTestEnv? Alternatively, what would cause a compilation error ？

Reply:
Removed. I forgot to remove it after refactoring the tests to use HummockTestEnv.